x-pack/plugins/security_solution/cypress/tasks/timelines.ts

```diff
@@ -19,7 +19,10 @@ export const exportTimeline = (timelineId: string) => {
 };
 
 export const openTimeline = (id: string) => {
-  cy.get(TIMELINE(id), { timeout: 500 }).click();
+  // This temporary wait here is to reduce flakeyness until we integrate cypress-pipe. Then please let us use cypress pipe.
+  // Ref: https://www.cypress.io/blog/2019/01/22/when-can-the-test-click/
+  // Ref: https://github.com/NicholasBoll/cypress-pipe#readme
+  cy.get(TIMELINE(id)).should('be.visible').wait(1500).click();
```

Comment:
NOTE: Tested and this should work as is, even though it is part of a skipped test. This will reduce flake for us for now until we integrate the pipe.